code/src/components/FifthQuestionMore.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import './fifthQuestion.css';
```

Comment:
its a good question as I was also thinking about the good rationale of this approach of creating separate css files. So far my rationale is that a component should be as independent as possible so it should have its own styling sheet also. in this way it could be bundle as a package and just to be reused in any other application.